controllers/userController.js

```diff
@@ -19,7 +19,7 @@ const signIn = async (req, res) => {
     try {
         const { email, password } = req.body;
         const token = await userService.signIn(email, password)
```

Comment:
```suggestion
        const { id, username, token } = await userService.signIn(email, password)
```
객체를 리턴 받게 되면 이렇게 구조 분해 할당할 수 있습니다.

Reply:
감사합니다!!